controllers/slice/namespaces.go

```diff
@@ -718,9 +719,19 @@ func (r *SliceReconciler) labelAppNamespaces(ctx context.Context, cfgAppNsList [
 		// label does not exists on namespace
 		namespace := &corev1.Namespace{}
 		err := r.Get(ctx, types.NamespacedName{Name: cfgAppNs}, namespace)
+		// If the namespace itself doesn't exist on specified cluster then first create it
 		if err != nil {
```

Comment:
Add additional check after err != nil to verify that error is indeed "Failed to find namespaces". 
Like 
`if errors.IsNotFound(err) { 
log.Info("Failed to find Namespaces" , "namespace", cfgAppNs)
.....`

https://pkg.go.dev/k8s.io/apimachinery/pkg/api/errors#IsNotFound ref for the function

Reply:
Addressed